core/workspace_svg.js

```diff
@@ -1078,25 +1106,15 @@ Blockly.WorkspaceSvg.prototype.resize = function() {
   if (this.flyout_) {
     this.flyout_.position();
   }
-  /** @type {Array<Blockly.IPositionable>} */
-  var positionableEls = [];
-  if (this.trashcan) {
-    positionableEls.push(this.trashcan);
-  }
-  if (this.zoomControls_) {
-    positionableEls.push(this.zoomControls_);
-  }
-  if (positionableEls) {
-    var metricsManager = this.getMetricsManager();
-    var viewMetrics = metricsManager.getViewMetrics();
-    var absoluteMetrics = metricsManager.getAbsoluteMetrics();
-    var toolboxMetrics = metricsManager.getToolboxMetrics();
-    var savedPositions = [];
-    for (var i = 0, uiElement; (uiElement = positionableEls[i]); i++) {
-      uiElement.position(
-          viewMetrics, absoluteMetrics, toolboxMetrics, savedPositions);
-      savedPositions.push(uiElement.getBoundingRectangle());
-    }
+
+  var positionables = /** @type {!Array<!Blockly.IPositionable>} */
```

Comment:
This case should no longer be necessary.

Reply:
just removed it, right as you commented